packages/react/tests/types.test.tsx

```diff
@@ -2,7 +2,7 @@
 import * as Stitches from '../types/index'
 import { createStitches } from '../types/index'
 
-const { css, global, keyframes, styled, theme } = createStitches({
+const { css, globalCss, keyframes, styled, theme } = createStitches({
```

Comment:
The test file was erroring because `global` doesn't exist so fixed this while I was here too.